### PR TITLE
models fixes + new texture

### PR DESCRIPTION
**Pull request by cloudziq**
fixes for:
  - wrong shading on door01
  - broken palmsy iqmpitch
  
added:
  - palmsy mask, for color attribute to affect only leafs

### CODE REVIEW

**Comment by cloudziq** [comment]
improved 'rusty set' of pipes to look a bit more properly with model color, closer one to the camera is how it looks after this change:

File: palmsy/iqm.cfg [hunk 1]
```diff
@@ -2,19 +2,21 @@
...
-iqmpitch * 1
+iqmpitch "basebone" 1
 
-mdlscale 100
+mdlscale 50
 mdlspec 10
 mdlbb 6 80
 mdlellipsecollide 1
+mdlcolor -1
 mdlyaw 90
```
Comment (qreeves) [inline palmsy/iqm.cfg]: No newline at end of file.

**Review by qreeves — CHANGES_REQUESTED** [review]

File: palmsy/iqm.cfg (hunk 1 above)
Comment (cloudziq) [reply palmsy/iqm.cfg]: done, I should be paying more attention to these ending newlines

Comment (Dziq84) [reply palmsy/iqm.cfg:22]: done, I should be paying more attention to these ending newlines

**Pull request merged**